### ISSUE TITLE
Make System.String interactive

### ISSUE THREAD

**Issue by rpetrusha** [dotnet/dotnet-api-docs#1137]
## Make System.String interactive

[System.String](https://docs.microsoft.com/dotnet/api/system.string) is a high page-view reference topic in reference hub format; that is, it contains encyclopedic documentation for the String class, along with numerous examples.

**Comment by SpaceShot** [comment]
I'm looking at the rest of the examples.  The examples using System.IO do not work in try.dot.net (I believe this is a security consideration, at least for now).  

Many of the cultural comparisons give different results than the results expected in the trailing comments in try.dot.net.  In some cases it was clear to me the try.dot.net result wasn't correct.  So I began double-checking them against the same sample running in .NET Core 2.2.  Simple console apps matched the output from the trailing comments.

In the end, I decided that there may be something wrong with cultural conversions in try.dot.net and I decided not to convert any of those samples, even if their results matched the trailing comments.  This could be by luck, and someone who decides to play with the sample might get confused later.

Beyond that, I am looking at the remaining samples to propose a PR to change the simplest conversions over to try.dot.net interactive.
